pkg/utils/subscription.go

```diff
@@ -1242,6 +1242,19 @@ func CreateClusterManagementAddon(clt client.Client) {
 				Description: "Processes events and other requests to managed resources.",
 				DisplayName: "Application Manager",
 			},
+			SupportedConfigs: []addonV1alpha1.ConfigMeta{
+				{
+					ConfigGroupResource: addonV1alpha1.ConfigGroupResource{
+						Group:    "addon.open-cluster-management.io",
+						Resource: "addondeploymentconfigs",
```

Comment:
We don't need add addondeploymentconfig here, we want user add this manfully when user want to enable the addondeploymentcofngis as the addon configuration